agent/src/main/java/org/apache/airavata/mft/agent/TransportMediator.java

```diff
@@ -82,140 +82,133 @@ public String transfer(TransferCommand command, Connector inConnector, Connector
 
 
         // Monitoring the completeness of the transfer
-        Thread monitorThread = new Thread(new Runnable() {
```

Comment:
Lots of unrelated changes. Do not do auto formatting from IDE and always look for the diff before committing.